docker/init

```diff
@@ -82,7 +82,6 @@ done
 update-ca-certificates
 
 # Further settings
-export PORTUS_PUMA_HOST="0.0.0.0:3000"
```

Comment:
I wouldn't change this, since it will break the deployment of most users. I propose to export this env. variable to this default value if none has been provided.

Reply:
This isn't going to help with my use-case: have puma bind to unix socket even when running Portus using the official container, see:

https://github.com/SUSE/Portus/blob/7358f3a81be062218cdacb209702fce089af2f23/config/puma.rb#L19-L31

Another way would be to rework the code snipped I pasted above to something like:

```ruby
if ENV["PORTUS_PUMA_HOST"] && !ENV["PORTUS_PUMA_USE_UNIX_SOCKET"]
...
else
# bind to socket
end
```

Reply:
I'd prefer the code you just pasted instead of the old one because I believe that telling people to always set this env. variable to "0.0.0.0:3000" for the most common use case is confusing. Instead, if you really know what you are doing, you can provide this extra env. variable.

So a :+1: from my side.

Reply:
Oh, and instead of `!ENV["PORTUS_PUMA_USE_UNIX_SOCKET"]`, I think it's safer `ENV["PORTUS_PUMA_USE_UNIX_SOCKET"].blank?`